cmd/upload.go

```diff
@@ -266,6 +266,7 @@ func uploadRun(cmd *cobra.Command, args []string) error {
 	release := &keygenext.Release{
 		ID:        uploadOpts.Release,
 		PackageID: &uploadOpts.Package,
+		ProductID: keygenext.Product,
```

Comment:
This can be removed since `release.Get()` already has access to the current product.